lib/services/cordova-plugins.ts

```diff
@@ -19,13 +19,13 @@ export class CordovaPluginsService implements ICordovaPluginsService {
 		this.$project.ensureCordovaProject();
 	}
 
-	public getPlugins(keywords: string[]): IPlugin[] {
+	public getPlugins(keywords: string[]): IShortInformationPlugin[] {
```

Comment:
Have you tested plugins fetch command after this change? I think it is using getPlugins method as well, so this will impact it as well.